rust-runtime/aws-smithy-types/test_data/date_time_format_test_suite.json

```diff
@@ -0,0 +1,1334 @@
+{
+  "format_date_time": [
```

Comment:
maybe "tests" would be a clearer name for this key?

Reply:
a `description` top level field explaining the fields and the expected test strategy would probably be good